unison-src/new-runtime-transcripts/codeops.md

```diff
@@ -104,7 +104,7 @@ f x = h zap x
 fVal : Value
 fVal = Value.value f
 
-fDeps : [Term]
+fDeps : [Link.Term]
```

Comment:
The longer name here is due to there being `Doc2.Term`, used for embedded terms.